src/style.css

```diff
@@ -1,3 +1,59 @@
 body {
+  width: 1200px;
```

Comment:
remove this width, because your header not stretching to full width on a large displays